windows/JackAtomic_os.h

```diff
@@ -30,12 +30,25 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
 #endif
 
 #ifndef inline
-	#define inline __inline
+    #if !defined(_M_X64)
```

Comment:
When is _M_X64 defined?

Reply:
this makes little sense, will research this again, even if I used _WIN64 I don't think it's a solution to the problem that left because of it on my computer.